honeybee_schema/radiance/properties.py

```diff
@@ -2,12 +2,13 @@
 from pydantic import Field, constr, validator, root_validator
 from typing import List, Union, Optional
 
-from .modifier import Plastic, Glass, BSDF, Glow, Light, Trans, Void
+from .modifier import Plastic, Glass, BSDF, Glow, Light, Trans, Void, Mirror
 from .modifierset import ModifierSet, ModifierSetAbridged
 from .._base import NoExtraBaseModel
 
 # Unioned Modifier Schema objects defined for type reference
-_REFERENCE_UNION_MODIFIERS = Union[Plastic, Glass, BSDF, Glow, Light, Trans, Void]
+_REFERENCE_UNION_MODIFIERS = Union[Plastic, Glass, BSDF, Glow, Light, Trans, Void,
+                                   Mirror]
```

Comment:
I would make this list once in a utility file and then import it to all these other modules. You already have it in 3 different places and it will be painful pretty soon to update all of them every time you make a change to modifiers.

Reply:
Sure. Do I need a separate utility file for it? I think I can just define it in the modifier.py class and import it from there.

Reply:
@mostaphaRoudsari I added the list to modifiers.py and imported from there. There's still a little bit of redundancy since the version that's used locally in modifiers.py has to be a list of strings rather then a list of objects. Let me know if you want it changed.

Reply:
Since we are using Python 3.7+ we can use the annotations option which means we don't need them to be strings. See the bottom of the page here: https://pydantic-docs.helpmanual.io/usage/postponed_annotations/#self-referencing-models

Reply:
Done! It looks like it works.